public/examples/ach.html

```diff
@@ -24,10 +24,11 @@
         return ach;
       }
 
-      async function createPayment(token) {
+      async function createPayment(token, idempotencyKey) {
         const body = JSON.stringify({
           locationId,
           sourceId: token,
+          idempotencyKey,
```

Comment:
```suggestion
          idempotencyKey: window.crypto.randomUUID(),
```

thoughts on generating here instead of passing through as an argument? I think it'd be functionally equivalent but could be overlooking something

Reply:
Wouldn't that mean that we generate it after the button is clicked? So clicking it twice means two separate idempotency keys.